src/implementations/vanilla/components/form-radio/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@ecl/vanilla-component-form-radio",
```

Comment:
do we need to keep the 'form' in the package name? It isn't the case for twig and specs packages. For the checkboxes it had been removed

Reply:
I removed the "form" part.